### ISSUE TITLE
Run tests without constantly restarting the process.

### ISSUE THREAD

**Issue by emilio** [servo/servo#14882]
@jgraham mentioned in https://github.com/servo/servo/issues/6160 that that was one of the blockers, but I don't see any other issue filled for this.

I guess this could be doable now?

cc @jdm @Ms2ger @asajeffrey @aneeshusa @larsbergstrom

**Comment by jgraham** [comment]
Yeah it seems worth trying again, although it isn't entirely clear if it will produce better performance or acceptable reliability. It should be possible to try wpt with `--product=servodriver` to see what regressed in the last 18 months or so.

**Comment by jdm** [comment]
Tests take about 13 minutes to run right now. Even a 50% reduction in time taken would not have a big impact on the overall time taken when building and testing, so I don't feel like investigating this is a big priority right now.
